### ISSUE TITLE
Work & install without admin right 

### ISSUE THREAD

**Issue by GeraudFabien** [dotnet/maui#125]
## Summary
It would be great to be able start developing from nothings without the need of admin right. It was that who make me stop using XF after 3 week with our IT to get it install. And the emulator didn't work because i was missing one right that my IT refuse to get me so we choose to stop using XF.

## API Changes
N/A

## Intended Use Case
- As a dev i want to start working without delay

## The dream
- As a dev i wish to Be able to download Zip binaries and just unzip and run.
- As a dev i wish to Be able to work with dotnet install also as binaries
- As a tester i wish to Be able to get a link where i can download emulator to test on my machine
- As a tester i wish to Be able to get a a container where i can execute test

**Comment by Happypig375** [comment]
This is really a Visual Studio issue. Visual Studio requires administrator permissions to install and update.

**Comment by Happypig375** [comment]
>Yes and no since even when you have VS you have to install XF. And since we didn't have the device for the start of the project i have to use the emultator. And the emulator need far more specific right to configure and some are hard to get like the right to be a hype-v administrator.

Xamarin.Forms is a _package_ that you can just install. Xamarin.iOS and Xamarin.Android are _frameworks_ that you install inside the Visual Studio installer. Android emulators are part of the Xamarin workload inside the Visual Studio Installer. So, this is a Visual Studio issue and not really related to Xamarin.Forms/MAUI.

**Comment by hartez** [comment]
> Xamarin.Forms is a _package_ that you can just install. Xamarin.iOS and Xamarin.Android are _frameworks_ that you install inside the Visual Studio installer. Android emulators are part of the Xamarin workload inside the Visual Studio Installer. So, this is a Visual Studio issue and not really related to Xamarin.Forms/MAUI.

All technically true, but it's still good feedback for us.

**Comment by GeraudFabien** [comment]
>But yes it's not a problem of XF but it's a real problem

And it's important to track this because there will be a VS code plugin and it would be great if it's easy to use.

And even with VS code they will have the problem since the issue was link to the way emulator work 
(or was working it was 2 year ago idk if it's still true). Because they need to automate the launch of Hyper-V VM. So even if there was a tool outside of VS it would still have the same problem. That why i propose container since it's one solution to bypass right problem and work seamlessly everywhere.

I also know that there may be no solution. I just open an issue on a big problem since i heard that i'm far from being the only one to hit this problems. It's just a problem to track.

**Comment by BretJohnson** [comment]
@GeraudFabien So we can help improve things, can you give some more specifics on what you had to go through here.

First, to clarify (and setting aside the emulator issue for the moment), do you not have Administrator access to your machine at all, so you can't install many programs like say Visual Studio. How did you get past that? By having IT install Visual Studio on your machine? In that case, I can see that having IT do the initial install maybe isn't so bad, but when Visual Studio requires administrator access after the initial install (which it sometimes does today) that could be annoying. We plan to make installing/updating the Android SDKs and virtual device images no longer require admin access - so I think that will help you here, but I wanted to make sure I understand all the issues.

And then on the emulator specifically, do you remember what the "one right that my IT refuse to get me" was? They had some policy in place around not allowing enabling Hyper-V, say? There may be some things we can do to help there too, but knowing more specifics would help. Thanks much for your feedback.

**Comment by GeraudFabien** [comment]
>  do you not have Administrator access to your machine at all, so you can't install many programs like say Visual Studio. How did you get past that? By having IT install Visual Studio on your machine? 
Yes i didn't install VS it was already install and i ask my SI to do the operation for me. Since it was new for them they ask me to help them even if i didn't have the right to touch my pc during the operation.

> but when Visual Studio requires administrator access after the initial install (which it sometimes does today) that could be annoying.
yes.

> They had some policy in place around not allowing enabling Hyper-V
We install hyper-V. And Xamarin was working for the IT. But i needed the role "Hyper-V admin" to used it too. And since 3 weeks pass to get to this point we decided to give up. But it was 2 years ago so i don't really remember exactly

The point of the issue is to track that configuring the environment for basic debug is simple and straight forward. Maybe my issue title is not right. It's more about manage the experience of a new user between searching for a course and debugging it's first sample.

It was on VS2013.

**Comment by IeuanWalker** [comment]
@BretJohnson I had the same issues when it comes to emulators. 
My IT department added a bunch of policies to get around them.

The main issue I had was that the emulator would try and create a new network adaptor. But there was a policy that stops new network adaptors, so a new policy was added to my account to allow me to have more.

Are IT are pretty restricted and everything that can be lockdown is, but if we need something unlocked, they are really helpful and unlock it for us.

So would be nice to have a list of stuff required, rather than troubleshooting with IT.


Another example of this was when I tried using Entity Framework, none of the commands where working, called IT turns out it was trying to execute a PowerShell file in a locked location, 10m later it was unlocked and everything worked.
